akka-docs/src/main/paradox/serialization-classic.md

```diff
@@ -0,0 +1,78 @@
+# Classic Serialization
+
+@@include[includes.md](includes.md) { #actor-api }
+
+Serialization is the same for Classic and Typed actors. It is described in @ref:[Serialization](serialization.md),
+aside from serialization of `ActorRef` that is described @ref:[here](#serializing-actorrefs).
+
+## Dependency
+
+To use Serialization, you must add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group="com.typesafe.akka"
+  artifact="akka-actor_$scala.binary_version$"
+  version="$akka.version$"
+}
+
+## Serializing ActorRefs
```

Comment:
this section was from original serialization.md, moved here without any changes apart from initial sentence.